### ISSUE TITLE
make all demo values valid

### ISSUE THREAD

**Issue by Morriz** [redkubes/otomi-core#173]
# User story
Now that we have encryption disabled by default we should be providing only valid values in `.demo/*` files.

All secret values are now the same "secretvalue". I believe this is to function as a marker, so please search for that word and see how it is used and if we can overcome the problem if we abandon that marker mechanism.

# Tasks

core:
- [ ] Execute Otomi bootstrap with demo values that work "out of the box".
- [ ] Find out where otomi-values uses a placeholder, and make it work for the demonstration. 
- [ ] Provide a diff of all the changes that were necessary to get a working values repo.
- [ ] Enable it to work with GCP.
- [ ] Record a screencast to demonstrate it to new users.

# Definition Of Done
- [ ] Tasks are done
- [ ] (Unit) Tests are performed
- [ ] (Architecture) Design Record(s) have been added as `adr/*.md` and appended to list in `adr/_index.md`
- [ ] Specs and demo files have been updated to reflect code changes
- [ ] Documentation has been updated
- [ ] Functionality, code, and/or documentation has been peer reviewed
- [ ] Relevant team members have been notified

**Comment by j-zimnowoda** [comment]
Good point, some charts has some constraints for value length. e.g.: recently I had to change oauth2-proxy cookieSecret

**Comment by Morriz** [comment]
> Good point, some charts has some constraints for value length. e.g.: recently I had to change oauth2-proxy cookieSecret

which was problematic, as you did not test the results of that, as the next deploy broke because of a faulty regex ;p

**Comment by Morriz** [comment]
@svatwork can you report the status of this issue? I would like to close it as its intent was to deliver demo values that don't need rework

**Comment by svatwork** [comment]
This is still open, it's on my to do list after #259 and #164.

**Comment by Morriz** [comment]
@svatwork If you park work that has deps, please give it label "on hold" and add those deps. Can you do that for this item as well?

**Comment by Morriz** [comment]
closing as too vague

**Comment by svatwork** [comment]
I still know what it's about, would you like me to refine it? I still think it's important.

**Comment by Morriz** [comment]
I think the values are good now, unless you can explicitly list ones that aren't?

**Comment by svatwork** [comment]
Well I think that's an assumption and I hope it's true of course. But it would give me some peace of mind if I can go through it once. But other than that, this story is also about giving a visual demonstration about the initial phase of starting out with Otomi, which can be very useful for new users.

**Comment by Morriz** [comment]
Ok, lets add issues found after creating a demo movie.
